client/elife-theme/src/elements/Button.js

```diff
@@ -0,0 +1,46 @@
+import { css } from 'styled-components'
+import { th, darken } from '@pubsweet/ui-toolkit'
+
+const extraSmall = css`
+  font-size: 11px;
+  line-height: calc(${th('gridUnit')} * 2);
+  padding: ${th('gridUnit')};
+`
+
+const small = css`
+  padding: ${th('gridUnit')};
+`
+
+const primary = css`
+  border-color: ${th('colorPrimary')};
+
+  &:focus,
+  &:hover {
+    border-color: ${darken('colorPrimary', 0.3)};
+  }
+
+  &:active {
+    border-color: ${darken('colorPrimary', 0.5)};
+  }
+
+  &[disabled] {
+    &:focus,
+    &:hover,
+    &:active {
+      border-color: ${th('colorPrimary')};
+    }
+  }
+`
+
+// TODO this should be updated when pubsweet changes gridUnit to 8px
+export default css`
+  line-height: calc(${th('gridUnit')} * 4);
+  min-width: calc(${th('gridUnit')} * 16);
+  padding: calc(${th('gridUnit')} * 2);
+  font-size: 14px;
```

Comment:
shouldnt this be `fontSizeBaseSmall`?

Reply:
these should be accessed from `theme.space`

Reply:
Check the discussion me & tam had that was resolved about font size. Regarding `theme.space`, can you point me to one place where this is specified? There was too much spam about this last week and now I'm confused. My take is that it should be documented in one place (e.g. github issue). Also, I initially wanted to use something to account for those size labels but my understanding was that we don't want it?

Reply:
For the record I'm still not convinced about the 14px.